template.html

```diff
@@ -2,24 +2,229 @@
+                            <div class="project-title">Summer House</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house5.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Brick House</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house2.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Renovated</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house3.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Barn House</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house4.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Summer House</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house2.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Brick House</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house5.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Renovated</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house4.jpg">
+                            </div>
+                        </li>
+                        <li class="project-item">
+                            <div class="project-title">Barn House</div>
+                            <div class="project-img">
+                                <img class="img" src="https://www.w3schools.com/w3images/house3.jpg">
+                            </div>
+                        </li>
+                    </ul>
+                </div>
             </div>
-        </div>
-        <header></header>
-        <main></main>
-        <footer></footer>
+            <div class="about" id="about">
```

Comment:
Try to work with the help of classes, at least in such cases and do code refactoring by deleting junk